monitoring-satellite/main.jsonnet

```diff
@@ -1,22 +1,18 @@
 // This file is used to update monitoring-satellites with ArgoCD
 local monitoringSatellite = (import './monitoring-satellite.libsonnet');
+local excludedComponents = [
+  'blackboxExporter',
+  'kubePrometheus',
+  'restrictedPodSecurityPolicy',
+];
 
 [
   monitoringSatellite.kubePrometheus.namespace,
   monitoringSatellite.kubePrometheus.prometheusRule,
   monitoringSatellite.restrictedPodSecurityPolicy,
 ] +
-[monitoringSatellite.kubeStateMetrics[name] for name in std.objectFields(monitoringSatellite.kubeStateMetrics)] +
-[monitoringSatellite.grafana[name] for name in std.objectFields(monitoringSatellite.grafana)] +
-[monitoringSatellite.prometheus[name] for name in std.objectFields(monitoringSatellite.prometheus)] +
-[monitoringSatellite.gitpod[name] for name in std.objectFields(monitoringSatellite.gitpod)] +
-[monitoringSatellite.alertmanager[name] for name in std.objectFields(monitoringSatellite.alertmanager)] +
-[monitoringSatellite.kubernetesControlPlane[name] for name in std.objectFields(monitoringSatellite.kubernetesControlPlane)] +
-[monitoringSatellite.nodeExporter[name] for name in std.objectFields(monitoringSatellite.nodeExporter)] +
-[monitoringSatellite.prometheusOperator[name] for name in std.objectFields(monitoringSatellite.prometheusOperator)] +
-[monitoringSatellite.certmanager[name] for name in std.objectFields(monitoringSatellite.certmanager)] +
-[monitoringSatellite.werft[name] for name in std.objectFields(monitoringSatellite.werft)]
-
-// Exposed by monitoring-satellite object, but we don't want to rollout yet:
-// [monitoringSatellite.otelCollector[name] for name in std.objectFields(monitoringSatellite.otelCollector)]
```

Comment:
If I remove the comments here and monitoring-satellite is rendered with `tracing_enabled == false`, otelCollector won't exist inside monitoringSatellite object, leading to error.